.github/workflows/publish-stack-set-to-service-catalog.yaml

```diff
@@ -104,3 +99,8 @@ jobs:
           Key=Owner,Value='platform-and-sre-pod@digital.cabinet-office.gov.uk' Key=Service,Value=dynatrace
         env:
           AWS_DEFAULT_REGION: eu-west-2
+      - name: Create new and delete old stack instances and delete old stack set
```

Comment:
Think this should go before the new stack creation. Once the new stack is ready , we can add the instances to new release stack.